test/e2e/middleware-param-rewrite-navigation/pages/a.js

```diff
@@ -0,0 +1,14 @@
+import { useRouter } from 'next/router'
+import Link from 'next/link'
+
+export default function IndexPage() {
+  const router = useRouter()
+  return (
+    <>
+      <span>{router.query.param}</span>
+      <Link href="/b">
+        <a>To "b"</a>
```

Comment:
```suggestion
        <a>To "a"</a>
```

Reply:
From page "a", you would navigate to "b", so this is correct. (Doesn't really matter though, we target the `a` tag to click.)

Reply:
Oh then the other page should be "to a" right? Currently both say "to b" which was a bit confusing when reproducing